examples/openspending/pages/[...slug].tsx

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import clientPromise from '@/lib/mddb';
+import { GetStaticProps } from 'next';
+import Layout from '../components/_shared/Layout';
+import ReactMarkdown from 'react-markdown';
+import { formatDate } from '@/utils/formatDate';
+import rehypeRaw from "rehype-raw";
+import matter from 'gray-matter'
+
+export default function Page({ source, meta }) {
+  return (
+    <Layout>
+      <article className="docs prose-a:text-primary dark:prose-a:text-primary-dark prose-strong:text-primary dark:prose-strong:text-primary-dark prose-code:text-primary dark:prose-code:text-primary-dark prose-headings:text-primary dark:prose-headings:text-primary-dark prose text-primary dark:text-primary-dark prose-headings:font-headings dark:prose-invert prose-a:break-words mx-auto p-6">
+        <header>
+          <div className="mb-4 flex-col items-center">
+            {meta.title && (
+              <h1 className="flex justify-center">{meta.title}</h1>
+            )}
+            {meta.date && (
+              <p className="text-sm text-zinc-400 dark:text-zinc-500 flex justify-center">
+                <time dateTime={meta.date}>{formatDate(meta.date)}</time>
+              </p>
+            )}
+          </div>
+        </header>
+        <section>
+          <ReactMarkdown rehypePlugins={[rehypeRaw]}>{source}</ReactMarkdown>
```

Comment:
If we use that, a big portion of the markdown files get broken, mostly because they have some sort of character thats not compatible with .mdx eg: "<" ">" if we are going to do that i would prefer to have a separate issue to fix all of the problems in the 600+ markdown files there

Reply:
Makes sense. I think we can create a new issue and tag it as "someday".